data/json/items/chemicals_and_resources.json

```diff
@@ -540,31 +540,73 @@
     "count" : 10
    },
    {
-    "type" : "AMMO",
+    "type" : "COMESTIBLE",
```

Comment:
It's sold as one today (though primarily intended for making cocktails and the like): https://en.wikipedia.org/wiki/Everclear_(alcohol)